book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ]As the project reviewer has earlier suggested, kindly add setters and getters for instance variables (remember about attr_accessor). Since the @rental variable is also an instance variable for the Book class, you should use attr_accessor as well to be able to get and set the rental variable 👍🏽